### ISSUE TITLE
Add UI support

### ISSUE THREAD

**Issue by oerix** [LesTR/homeassistant-minimalistic-area-card#50]
### What happened?

It would be great if you could add some sort of UI support on the frontend, instead of having to use yaml to edit the card.
Thank you

### Card Version

1.2.9

### HomeAssistant Version

2024.8.3

**Comment by LesTR** [comment]
I agree, but I'm not good with UI stuff. I need to think about how it should look. Do you have any suggestions?

**Comment by LiquidPT** [comment]
I'd be willing to chime in on this. I was going to do it on the original repo, but saw that this fork is much more maintained

**Comment by LesTR** [comment]
@LiquidPT, have you made some progress on this? If not, I'll try to do something. I just don't want to duplicate the effort.

**Comment by LiquidPT** [comment]
> @LiquidPT, have you made some progress on this? If not, I'll try to do something. I just don't want to duplicate the effort.

I haven't. I'm just now getting time to get my head around how card projects are set up and work. The ones built into HA get fancy and do more abstraction, but I'm not sure if cards in HACS can leverage that code yet or not.

**Comment by LesTR** [comment]
I just released the first enhancement in this area - the editor uses a simple YAML component. This adds the basic logic and also allows to specify visibility and layout via HA UI. Let's iterate on this, and any feedback is welcomed.

**Comment by RkcCorian** [comment]
@LesTR : I downloaded v1.20.0. Strangewise I cannot add the card via the UI (via "+" symbol) anymore as it will not come up. I restarted HA already. Is this intended?

**Comment by LesTR** [comment]
>  Is this intended?

@RkcCorian  No, thanks for the report! I know where the issue came from and will prepare the fix soon.  Tracked in https://github.com/LesTR/homeassistant-minimalistic-area-card/issues/175

**Comment by LesTR** [comment]
Hehe, the card was mentioned in https://youtu.be/Y6YVQL0YHUw?t=57 Hopefully it will be merged into HACS soon.
